src/components/HistoryModal/Content.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {
+  Loading,
+  Table,
+  Text,
+  Button,
+  Container,
+  Col,
+  Spacer,
+  Tooltip,
+} from '@geist-ui/react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { AmmDexOperation, AmmOrder, RefundOperation } from 'ergo-dex-sdk';
+import { toast } from 'react-toastify';
+import { exploreTx } from '../../utils/redirect';
+import { isRefundableOperation } from '../../utils/ammOperations';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt, faUndo } from '@fortawesome/free-solid-svg-icons';
+import { useToggle } from '../../hooks/useToggle';
+import { ConfirmRefundModal } from '../ConfirmRefundModal/ConfirmRefundModal';
+import { truncate } from '../../utils/string';
+
+function renderOrder(
+  { status, txId, order }: AmmOrder,
+  open: boolean,
+  handleOpen: () => void,
+  handleClose: () => void,
+) {
+  return {
+    status,
+    txId: (
+      <CopyToClipboard text={txId} onCopy={() => toast.info('Copied')}>
+        <span style={{ cursor: 'pointer' }}>{truncate(txId)}</span>
+      </CopyToClipboard>
+    ),
+    operation: (
+      <Container>
+        <Col>
+          <Tooltip text={'View on Explorer'} type="dark">
+            <Button
+              icon={<FontAwesomeIcon icon={faExternalLinkAlt} />}
+              auto
+              size="small"
+              onClick={() => exploreTx(txId)}
+            />
+          </Tooltip>
+        </Col>
+        {isRefundableOperation(status) && (
+          <>
+            <Spacer x={0.2} />
+            <Col>
+              <Tooltip text={'Refund transaction'} type="dark">
+                <Button
+                  auto
+                  size="small"
+                  onClick={handleOpen}
+                  icon={<FontAwesomeIcon icon={faUndo} />}
+                />
+              </Tooltip>
+            </Col>
+            <ConfirmRefundModal txId={txId} open={open} onClose={handleClose} />
```

Comment:
Probably nouns would be better for the names of react components. So I suggest to rename `ConfirmRefundModal` to `RefundConfirmationModal`